src/mlpack/methods/ann/layer/batch_norm.hpp

```diff
@@ -185,19 +213,33 @@ class BatchNorm : public Layer<InputType, OutputType>
   //! mean and variance.
   double averageFactor;
 
+  //! Locally-stored input dimension.
```

Comment:
```suggestion
  //! Locally-stored number of input dimensions that we are applying batch normalization over.  (This is the product of this->inputDimensions from index 0 to (maxAxis - 1)).
```

I know that line is too long, but I can't reflow it correctly on my phone (sorry about that).